### DISCUSSION TITLE
wouldnt it be good to be able to select the version of openai in the configuration?

### DISCUSSION THREAD

**Discussion by JimiSweden** [arminbro/generate-react-cli#95] [General]
When looking at the OpenAI integration and models available you can read 
"gpt-3.5-turbo"  Most capable GPT-3.5 model and optimized for chat **at 1/10th the cost of text-davinci-003**. Will be updated with our latest model iteration."  https://platform.openai.com/docs/models/gpt-3-5

I think it sould be nice to be able to select the version to use for code generation in .env.local.
from my understanding the text-davinci-003 model might be better for ceode completion but being able to select model depending on how much I want to pay (more or less) would be nice.
`OPENAI_MODEL="text-davinci-003|gpt-4|gpt-3.5-turbo|..."`

//Jimi

**Comment by arminbro** [comment]
Hi @JimiSweden,

That's a great idea. When I initially integrated open AI with GRC, the DaVinci-003 model was the latest and the most capable code generator.

I'll look into adding a new `openAi` config object within the `generate-react-cli.json` config file, allowing developers to configure their openAi language model as a start.

Thanks for trying out GRC-openAi-alpha and providing your feedback. I appreciate it.

Thanks,
Armin
